lcls_naming_tool/lcls_naming_tool.py

```diff
@@ -31,11 +31,13 @@
 import argparse
 import subprocess
 import sys
+import os
 import json
 import re
-from pathlib import Path
 
 
+lcls_taxons_cfg = str(os.environ.get('LCLS_NAMING_TOOL_TAXONS_DIR'))
```

Comment:
You probably want to assign a backup directory for users who don't have this environment variable set. I suggest using the local taxons dir here as the backup.